src/main/java/com/wojteknier03/clinic_medical/mapper/UserMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.wojteknier03.clinic_medical.mapper;
+
+import com.wojteknier03.clinic_medical.dto.UserDto;
+import com.wojteknier03.clinic_medical.model.AppUser;
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface UserMapper {
+
+    UserDto userToUserDto(AppUser user);
+    AppUser userDtoToUser(UserDto userDto);
+
+    default List<UserDto> toDtoList(List<AppUser> user){
```

Comment:
nie piszemy implementacji jesli chcesz metode mapujaca liste po prostu zdefiniuj jej sygnature i tyle. Nie pisz implementacji to mapstruct ma nam implementacje wygenerowac